pkg/api/arc.yaml

```diff
@@ -474,6 +474,46 @@ components:
               "txid": "3d7770a2c3bbf890fe69ad33faadd3efb470b60d05b071eaa86e6597d480e111",
               "txStatus": "DOUBLE_SPEND_ATTEMPTED"
             }
+    
+    BatchedCallbacks:
+      type: object
+      description: batched callbacks object
+      required:
+        - count
+      properties:
+        count:
+          type: integer
+          description: number of callbacks in response
+        callbacks:
+          type: array
+          nullable: true
+          items:
+            $ref: '#/components/schemas/Callback'
+      example:
+        - batched:
```

Comment:
What is this `batched` property. I appears in `arc.json` line 734 as `"batched": null,`
Can it be removed?

Reply:
I based it on examples from the Callback section. I fixed it